src/components/StrategyTradesTable/StrategyTradesTable.helpers.js

```diff
@@ -3,73 +3,59 @@ import { saveAs } from 'file-saver'
 import _map from 'lodash/map'
 import _split from 'lodash/split'
 import _replace from 'lodash/replace'
+import _isEmpty from 'lodash/isEmpty'
+import _reduce from 'lodash/reduce'
 import { getPairFromMarket } from '../../util/market'
+import { getTradesHeaders, getPositionsHeaders } from './TradesTable/TradesTable.helpers'
 
 const getExportFilename = (prefix, extension = 'zip') => {
   // turn something like 2022-02-22T12:55:03.800Z into 2022-02-22T12-55-03
   const date = _replace(_split(new Date().toISOString(), '.')[0], /:/g, '-')
   return `${prefix}-${date}.${extension}`
 }
 
-const onTradeExportClick = (rawTrades, results, activeMarket, t, getCurrencySymbol) => {
-  const {
-    nCandles,
-    nTrades,
-    nGains,
-    nLosses,
-    nStrategyTrades,
-    nOpens,
-    pl,
-    pf,
-    maxPL,
-    minPL,
-    fees,
-    vol,
-    stdDeviation,
-    avgPL,
-  } = results
+const onTradeExportClick = (rawPositions, activeMarket, t, getCurrencySymbol) => {
+  const tHeaders = getTradesHeaders(t)
+  const pHeaders = getPositionsHeaders(t)
 
-  const tHeaders = {
-    price: t('table.price'),
-    amount: t('table.amount'),
-    pl: t('table.pl'),
-    label: t('table.label'),
-    mts: t('table.time'),
-    fee: t('table.fee'),
-  }
+  const positions = _map(rawPositions, ({
+    amount, entryPrice, closingPrice, entryAt, closedAt, id, pl,
+  }) => ({
+    [pHeaders.id]: id,
+    [pHeaders.entryAt]: _replace(new Date(entryAt).toLocaleString(), ',', ''),
+    [pHeaders.closedAt]: _replace(new Date(closedAt).toLocaleString(), ',', ''),
+    [pHeaders.entryPrice]: entryPrice,
+    [pHeaders.closingPrice]: closingPrice,
+    [pHeaders.amount]: amount,
+    [pHeaders.pl]: pl,
+  }))
+
+  const rawTrades = _reduce(rawPositions, (acc, position) => {
+    const { trades } = position
+
+    if (_isEmpty(trades)) {
+      return acc
+    }
+
+    return [...acc, ...trades]
+  }, [])
 
   const trades = _map(rawTrades, ({
-    price, amount, pl: tPl, fee, label, mts,
+    amount, order_id: orderID, order_js: order,
   }) => ({
-    [tHeaders.price]: price,
+    [tHeaders.id]: orderID,
+    [tHeaders.action]: amount < 0 ? 'SELL' : 'BUY',
+    [tHeaders.type]: order.type,
+    [tHeaders.timestamp]: new Date(order?.mtsCreate).toLocaleString(),
+    [tHeaders.executedAt]: new Date(order?.mtsUpdate).toLocaleString(),
+    [tHeaders.orderPrice]: order.price,
+    [tHeaders.tradePrice]: order.priceAvg,
     [tHeaders.amount]: amount,
```

Comment:
I also meant to get values also from common helper. Because if we see for trades, export is using `amount` key but UI renderer is retrieving from `trade?.order_js?.amountOrig || trade?.order_js?.amount` so that will create mismatch in data we show on UI and in exported file.
anyway I am merging this PR, @Blaumaus could you resolve above in separate PR ?